auth/auth.go

```diff
@@ -0,0 +1,42 @@
+package auth
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+
+	"github.com/ONSdigital/go-ns/rchttp"
+	"github.com/ian-kent/go-log/log"
+)
+
+func CheckServiceIdentity(ctx context.Context, zebedeeURL, serviceAuthToken string) error {
+	client := rchttp.DefaultClient
+
+	path := fmt.Sprintf("%s/identity", zebedeeURL)
+
+	var URL *url.URL
```

Comment:
url should not be exported

Reply:
It won't be exported as its declared in a function?